feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -262,6 +266,13 @@ private fun ContentScreen(
                 onStopVideoRecording = onStopVideoRecording,
                 zoomLevelDisplayState = remember { ZoomLevelDisplayState(isDebugMode) }
             )
+
+            DebugOverlayComponent(
+                toggleIsOpen = onToggleDebugOverlay,
+                previewUiState = previewUiState,
```

Comment:
could we not just directly pass the `debugUiState`?

Reply:
at l:199, previewUiState.zoomRatio is needed as the denominator to calculate the new absolute zoomRatio, so we have to pass in previewUiState